website/docs/r/system_center_virtual_machine_manager_server.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "System Center Virtual Machine Manager"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_system_center_virtual_machine_manager_server"
+description: |-
+  Manages a System Center Virtual Machine Manager Server.
+---
+
+# azurerm_system_center_virtual_machine_manager_server
+
+Manages a System Center Virtual Machine Manager Server.
+
+~> **Note:** By request of the service team the provider no longer automatically registering the `Microsoft.ScVmm` Resource Provider for this resource. To register it you can run `az provider register --namespace Microsoft.ScVmm`.
+
```

Comment:
Since this resource depends on some dependencies like an existing SCVMM environment and [Azure Arc resource bridge](https://learn.microsoft.com/en-us/azure/azure-arc/resource-bridge/overview), etc. Could you add a similar `note` like what [azurerm_arc_kubernetes_cluster](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/website/docs/r/arc_kubernetes_cluster.html.markdown#azurerm_arc_kubernetes_cluster) does to illustrate that the deployment of these dependencies is outside scope of this document and meanwhile provide some MS official guidance links of how to deploy these resources?

Reply:
I will confirm with service team about the related MS official guidance links.